virttest/utils_conn.py

```diff
@@ -603,6 +603,8 @@ def conn_setup(self):
         ssh_copy_id = self.SSH_COPY_ID
         ssh_agent = self.SSH_AGENT
         shell = self.SHELL
+        if rTimeout < 0:
+            rTimeout = 10
```

Comment:
If you want to check the value,  I think you should deal with any invalid value. Or just pass it to handle_prompts